.github/workflows/sync-release-branches.yml

```diff
@@ -0,0 +1,31 @@
+name: Keep release-* branches in sync with master
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```

Comment:
Can we use fedora:latest? or UBI latest ;) , you know keeping it Red

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer
I don't think we can :(

Reply:
I don't understand why we couldn't based on that documentation, what I am missing?

Reply:
wrong anchor, sorry:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

```
Available GitHub-hosted runner types are:

-- | --
Windows Server 2019 | windows-latest or windows-2019
Ubuntu 20.04 | ubuntu-latest (see note) or ubuntu-20.04
Ubuntu 18.04 | ubuntu-latest (see note) or ubuntu-18.04
Ubuntu 16.04 | ubuntu-16.04
macOS Big Sur 11.0 | macos-11.0
macOS Catalina 10.15 | macos-latest or macos-10.15


```

Reply:
And couldn't we use 
```yaml
jobs:
  my_job:
    container:
      image: fedora:latest
```
? if not I am ok, is just a personal preference